models/dish/dishValidation.js

```diff
@@ -0,0 +1,25 @@
+import Joi from 'joi';
+
+const dishValidationSchema = Joi.object({
+  name: Joi.string().required(),
+  owner: Joi.string().required(),
```

Comment:
it's better to create ObjectId string for Joi
something like that: `Joi.string().regex(/^[0-9a-fA-F]{24}$/)`

Reply:
Also, mongoose had its own validation method "isValidObjectId()". We can use this method :)

Reply:
Or a third option, remove this field from the schema, as it will be set on the backend and the frontend cannot change it.

Reply:
And I think it would be not bad to add id validation middleware.